packages/block/src/header.ts

```diff
@@ -518,13 +518,13 @@ export class BlockHeader {
       )
       throw new Error(msg)
     }
-    if ('cliqueActiveSigners' in blockchain === false) {
+    if ('cliqueActiveSigners' in (blockchain as any).consensus === false) {
       const msg = this._errorMsg(
         'PoA blockchain requires method blockchain.cliqueActiveSigners() to validate clique difficulty'
       )
       throw new Error(msg)
     }
-    const signers = (blockchain as any).cliqueActiveSigners()
+    const signers = (blockchain as any).consensus.cliqueActiveSigners()
```

Comment:
can TS infer that cliqueActiveSigners is available after the `in` check above? if not, at least cast as CliqueConsensus

Reply:
Ryan and I tried out a couple ideas but couldn't get it to work without introducing a circular dependency between `block` and `blockchain`. Decided to leave it as is for now.